relay-event-schema/src/protocol/contexts/memory_info.rs

```diff
@@ -0,0 +1,168 @@
+use relay_protocol::{Annotated, Empty, FromValue, IntoValue, Object, Value};
+
+use crate::processor::ProcessValue;
+
+/// Memory Info Context
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+pub struct MemoryInfoContext {
+    /// Currently allocated memory in bytes.
+    pub allocated_bytes: Annotated<u64>,
+
+    /// Boolean indicating if memory was compacted.
+    pub compacted: Annotated<bool>,
+
+    /// Boolean indicating if concurrent garbage collection occurred.
+    pub concurrent: Annotated<bool>,
+
+    /// Number of objects awaiting finalization.
+    pub finalization_pending_count: Annotated<u64>,
+
+    /// Fragmented memory that cannot be used in bytes.
+    pub fragmented_bytes: Annotated<u64>,
+
+    /// Total heap size in bytes.
+    pub heap_size_bytes: Annotated<u64>,
+
+    /// Threshold for high memory load detection in bytes.
+    pub high_memory_load_threshold_bytes: Annotated<u64>,
+
+    /// GC generation index.
+    pub index: Annotated<u64>,
+
+    /// Current memory load in bytes.
+    pub memory_load_bytes: Annotated<u64>,
+
+    /// Array of GC pause durations in milliseconds.
+    pub pause_durations: Annotated<Vec<Annotated<u64>>>,
```

Comment:
```suggestion
    pub pause_durations: Annotated<Arrayu64>>,
```

Also need to import that, same place where `Object` is from.